### PR TITLE
docs: review bt mesh models

### PR DESCRIPTION
**Pull request by greg-fer**
Ensure consistency of mesh model docs.

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

File: include/bluetooth/mesh/gen_prop_srv.rst [hunk 1]
```diff
@@ -3,59 +3,61 @@
 Generic Property Servers
 ########################
 
-A Generic Property Server holds a list of Device Properties.
-There are four different property servers:
+A Generic Property Server holds a list of device properties.
+The following different property servers are available:
```
Comment (trond-snekvik) [inline include/bluetooth/mesh/gen_prop_srv.rst:7]: This change was a bit awkward I think. I get the motivation behind removing the count, but in this case, it's very unlikely that it'll ever change. I think the old text was better, but perhaps removing "different" would be an okay middle ground?

File: include/bluetooth/mesh/gen_prop_srv.rst [hunk 2]
```diff
@@ -3,59 +3,61 @@
...
-- **Generic Client Property Server**: Provides a list of all properties supported by the Generic Property Client on the same element.
+- Generic Manufacturer Property Server - Provides access to Manufacturer properties, which are read-only properties set by the device manufacturer.
+- Generic Admin Property Server - Provides access to Admin properties, which can be read and written to.
+- Generic User Property Server - Provides access to both Admin and Manufacturer properties on its element, but only if the Property owner allows it.
+- Generic Client Property Server - Provides a list of all properties supported by the Generic Property Client on the same element.
   The Client Property Server does not actually own any properties, and does not need the user to provide access to any property values.
 
-Typically, only the administrator of the mesh network should have access to the Admin or Manufacturer servers, all other mesh nodes should operate on the User server.
-The Manufacturer and Admin servers may change the User property access rights for each individual Property it owns at runtime.
+When developing, take into account the following information about the servers:
```
Comment (trond-snekvik) [inline include/bluetooth/mesh/gen_prop_srv.rst:15]:
```suggestion
When developing, you should take the following into account:
```

File: include/bluetooth/mesh/gen_prop_srv.rst [hunk 3]
```diff
@@ -3,59 +3,61 @@
...
 
 Extended models
-================
+===============
 
 The Generic Admin Property Server and Generic Manufacturer Property Server should both extend the Generic User Property Server.
 This is not handled automatically by the model implementation.
-For this reason, a Generic User Property Server must be instantiated in the same element in the device composition data as the Generic Admin or Generic Manufacturer Property server.
-This ensures compliance with the Bluetooth Mesh Model specification.
+For this reason, to ensures compliance with the Bluetooth Mesh Model specification, a Generic User Property Server must be instantiated in the same element in the device composition data as the Generic Admin or Generic Manufacturer Property server.
```
Comment (trond-snekvik) [inline include/bluetooth/mesh/gen_prop_srv.rst:51]:
```suggestion
For this reason, to ensure compliance with the Bluetooth Mesh Model specification, a Generic User Property Server must be instantiated in the same element in the device composition data as the Generic Admin or Generic Manufacturer Property server.
```

File: include/bluetooth/mesh/light_ctrl_srv.rst [hunk 4]
```diff
@@ -90,19 +92,20 @@ When the Light LC Server is turned on, the following sequence takes place:
...
+The state machine cycles through the states with the following events:
 
 Timeout
-   The current state has timed out, and will automatically move into the next state.
+   Triggered when the current state has timed out.
+   The state machine automatically moves into the next state.
 
 On
-   A motion sensor is triggered or a light switch is pressed.
+   Toggled when a motion sensor is triggered or a light switch is pressed.
```
Comment (trond-snekvik) [inline include/bluetooth/mesh/light_ctrl_srv.rst:102]: Align with text below?
ie

```suggestion
   Toggled when a motion sensor is triggered or the On button is pressed on a light switch.
```

File: include/bluetooth/mesh/light_ctrl_srv.rst [hunk 5]
```diff
@@ -139,32 +146,32 @@ On event
...
-* The :ref:`bt_mesh_light_ctrl_cli_readme` model -- which should publish Light OnOff Set messages to the Light LC Server.
+  * The :ref:`bt_mesh_onoff_cli_readme` model -- which should publish to the Light LC Server's extended Generic OnOff Server model.
+  * The :ref:`bt_mesh_light_ctrl_cli_readme` model -- which should publish Light OnOff Set messages to the Light LC Server.
 
-Occupancy sensors may also trigger the On event, depending on the current state and occupancy mode:
+* Occupancy sensors can also trigger the On event, depending on the current state and occupancy mode:
 
-* If the occupancy mode is enabled, sensor readings that indicating activity can trigger an On event at any time.
-* If the occupancy mode is disabled, sensors cannot turn the lights on, but they will still prevent lights from turning off.
+  * If the occupancy mode is enabled, sensor readings that indicating activity can trigger an On event at any time.
```
Comment (trond-snekvik) [inline include/bluetooth/mesh/light_ctrl_srv.rst:156]:
```suggestion
  * If the occupancy mode is enabled, sensor readings that indicate activity can trigger an On event at any time.
```

File: include/bluetooth/mesh/lightness_srv.rst [hunk 6]
```diff
@@ -8,58 +8,54 @@ It should be instantiated in the light fixture node.
...
-The Light Lightness Server only provides write access to the Light state for the user, in addition to read access to all the meta states.
-
-The Light Lightness Setup Server provides write access to the Default Light and Light Range meta states.
-This allows the configurator devices to set up the range and the default value for the Light state.
+* The Light Lightness Server only provides write access to the Light state for the user, in addition to read access to all the meta states.
+* The Light Lightness Setup Server provides write access to the Default Light and Light Range meta states.
+  This allows the configurator devices to set up the range and the default value for the Light state.
 
 States
 ======
```
Comment (trond-snekvik) [inline include/bluetooth/mesh/lightness_srv.rst:22]: This section is missing the common "this model contains the following states" text

File: include/bluetooth/mesh/sensor_models.rst [hunk 7]
```diff
@@ -3,13 +3,18 @@
...
    :maxdepth: 1
    :glob:
 
    sensor_srv.rst
    sensor_cli.rst
+
+Common types
+============
+
+The Sensor models have no common types and only use their native types.
```
Comment (trond-snekvik) [inline include/bluetooth/mesh/sensor_models.rst:20]: Same as DTT models

File: doc/nrf/templates/mesh_model_main_README.rst [hunk 8]
```diff
@@ -32,7 +32,8 @@ Common types*
 
 .. tip::
    The Common types section is optional.
-   If the model does not have its own common types, do not add this section, but still mention at the end of the page that the model does not have its own common types.
+   If the model does not have its own common types, mention in this section that the model does not have its own common types.
```
Comment (mia-ko) [inline doc/nrf/templates/mesh_model_main_README.rst:35]:
```suggestion
   Use this section to inform if the model does not have its own common types.
```

**Comment by greg-fer** [comment]
@joerchan , can you have a look? You're a codeowner for this PR. Thanks!

**Pull request merged**